about.html

```diff
@@ -1,73 +1,86 @@
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+    <div class="container-about-page">
+        <header class="nav-container">
+            <nav class="top-content-nav">
+                <img src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\lambda-black.png">
+                <a href="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\index.html">Home</a>
+                <a href="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\about.html">About</a>
+                <a href="#products">Products</a>
+                <a href="#blog">Blog</a>
+                <a href="#contact">Contact</a>
+            </nav>
+        </header>
+        <div class="top-content">
+            <img class="image" src="C:\Users\coino\User-Interface\Sprint-Challenge--User-Interface\img\jumbo-about.png">
+            <div class="about-text-container">
+                <h2>About</h2>
+                <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view
+                    of disruptive innovation via workplace diversity and empowerment.</p>
+            </div>
+        </div>
+        <section class="top-content">
+            <div class="text-container">
+                <img src="img/about-plan.png" alt="strategy">
+                <h2>Strategy</h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <button><strong>Learn More</strong></button>
+            </div>
+
+            <div class="text-container">
+                <img src="img/about-working.png" alt="strategy">
```

Comment:
This is the correct way to source an image here on line 44. This is how it should be done throughout.